service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipApiService.java

```diff
@@ -265,8 +250,12 @@ public BipFileUploadResp uploadEvidenceFile(
       HttpEntity<MultiValueMap<String, Object>> httpEntity = new HttpEntity<>(body, headers);
 
       ResponseEntity<String> bipResponse =
-          restTemplate.exchange(url, HttpMethod.POST, httpEntity, String.class);
+          ceRestTemplate.exchange(url, HttpMethod.POST, httpEntity, String.class);
       BipFileUploadResp resp = new BipFileUploadResp();
+      log.info(
+          "bip response for upload: status: {}, message: {}",
```

Comment:
We need a claim or collection Id in the message to know which claim it is referring to.

Reply:
BIP upload API doesn't return a claim ID. A list of claim IDs can be put in the payload (see associatedClaimIds in BipFileProviderData) when calls the API